### PR TITLE
Remove get_cache() from controller

### PR DESCRIPTION
**Pull request by kmws**
(This is a part of #213)

Remove one layer of indirection.

### CODE REVIEW

**Comment by kmws** [comment]
Forgot about tests :s will fix them in the afternoon.

File: include/controller.h [hunk 1]
```diff
@@ -148,7 +143,7 @@ class controller {
 
 	ConfigPaths configpaths;
 
-	Reloader reloader;
+	std::unique_ptr<Reloader> reloader;
```
Comment (Minoru) [inline include/controller.h]: Sorry, I don't understand why you could possibly make that change. Can you explain?

File: include/view.h [hunk 2]
```diff
@@ -147,6 +148,8 @@ class view {
 	void cancel_input(std::shared_ptr<formaction> fa);
 	void delete_word(std::shared_ptr<formaction> fa);
 
+	cache* rsscache;
```
Comment (Minoru) [inline include/view.h]: On a principle that all new code should be top-notch, please initialize that pointer to `nullptr`.

File: include/controller.h (hunk 1 above)
Comment (kmws) [reply include/controller.h]: Hmm. The other way to do this I guess is define a `operator=` for Reloader and then re-initialize it after `rsscache` creation. Is that what you had in mind? Am I missing something?

Comment (Minoru) [reply include/controller.h]: Oh, sorry, I assumed `cache` is a non-pointer field of `controller`, and thought you could initialize `Reloader` in `controller`'s member initialization list. My bad.

**Comment by Minoru** [comment]
Cool! I'll merge once Travis build is done.

**Comment by Minoru** [comment]
Thank you very much, @kmws!

**Pull request merged**